### ISSUE TITLE
Page trim leaving blank spaces between pages - A4 pattern not joined correctly

### ISSUE THREAD

**Issue by lucielavickova** [cfcurtis/pdfstitcher#149]
the old bug https://github.com/cfcurtis/pdfstitcher/issues/131 reappeared :(
app version: v0.6.1

When trying to trim the document margins and choosing the "overlap" ("Překrytí" in czech) option, the document is created successfully and the margins are removed, but there are still blank spaces between pages:
![image](https://user-images.githubusercontent.com/16803915/167255119-a951b319-afd1-4fa6-815c-1f34af975358.png)

pdfstitcher settings:
![image](https://user-images.githubusercontent.com/16803915/167255495-1194f640-7593-4ed1-856b-e7fac78da363.png)

for easier debugging, check out the document: https://1drv.ms/b/s!Al4bp4wU8L5IickQW7sizVhlm5oGGQ?e=gSNheO

**Comment by cfcurtis** [comment]
Argh, bugs sneaking back in! I'm guessing this might be related to #148 as well. I will look into these for the next bug fix release.

**Comment by cfcurtis** [comment]
I think this one might be some white borders on the PDF itself. If I set the trim to something big like 1", I can see the pages overlapping as expected, but there is still always a white border. However, I have fixed the issue with the trim vs overlap, so I hope to get that out soon (this PDF has a page rotation of 360 degrees, which is a case I was not handling!).

![image](https://user-images.githubusercontent.com/282731/167520232-7d2490e1-28ed-49f4-9ea7-2a49ff46c5e2.png)
